vendor.mod

```diff
@@ -66,6 +66,7 @@ require (
 	github.com/cenkalti/backoff/v4 v4.3.0 // indirect
 	github.com/cespare/xxhash/v2 v2.3.0 // indirect
 	github.com/containerd/log v0.1.0 // indirect
+	github.com/cpuguy83/go-md2man/v2 v2.0.6 // indirect
```

Comment:
We could make use of the new `tool `directive starting Go 1.24:
* https://go.dev/doc/modules/managing-dependencies#tools
* https://go.dev/ref/mod#go-mod-file-tool

Reply:
Ah, yes, but we're not on go1.24 yet 😅 

At least currently it works, so I guess it's fine with how we have it right now?